librosa/segment.py

```diff
@@ -1138,3 +1176,77 @@ def path_enhance(
         np.clip(R_smooth, 0, None, out=R_smooth)
 
     return R_smooth
+
+
+def __affinity_bandwidth(rec, bw_mode, k):
+
+    # the api allows users to specify a scalar bandwidth directly, besides the string based options.
+    if isinstance(bw_mode, (int, float)):
+        bandwidth = bw_mode
+        if bandwidth <= 0:
+            raise ParameterError(
+                "Invalid scalar bandwidth={}. " "Must be strictly positive.".format(bandwidth)
+            )
+        return bandwidth
+
+    if bw_mode is None:
+        bw_mode = 'med_k_scaler'
+
```

Comment:
should be `scalar` not `scaler`

Reply:
Thanks. Will fix all instance of this typo.